neo/Consensus/ConsensusContext.cs

```diff
@@ -336,7 +354,7 @@ public ConsensusPayload MakePrepareResponse()
             });
         }
 
-        public void Reset(byte viewNumber)
+        public void Reset(byte viewNumber, bool resetFutures = true)
```

Comment:
This is always true, but I don't see the code calling it with the false value.

Reply:
No, it is not always true.